examples/slider/range-temperature.html

```diff
@@ -0,0 +1,215 @@
+
+  <!--  Core js and css shared by all examples; do not modify when using this template. -->
+  <link rel="stylesheet" href="https://www.w3.org/StyleSheets/TR/2016/base.css">
+  <link rel="stylesheet" href="../css/core.css">
+  <script src="../js/examples.js"></script>
+  <script src="../js/highlight.pack.js"></script>
+  <script src="../js/app.js"></script>
+
+  <link rel="stylesheet" href="css/range-temperature.css">
+  <script src="js/range-temperature.js"></script>
+</head>
+<body>
+  <nav aria-label="Related Links" class="feedback">
+    <ul>
+      <li><a href="../../#browser_and_AT_support">Browser and Assistive Technology Support</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/issues/new">Report Issue</a></li>
+      <li><a href="https://github.com/w3c/aria-practices/projects/3">Related Issues</a></li>
+      <li><a href="../../#slider">Design Pattern</a></li>
+    </ul>
+  </nav>
+  <main>
+    <h1>HTML Range Examples with aria-orientation and aria-valuetext</h1>
+    <p>
+      The following example of a temperature range slider demonstrating the using <code>aria-valuetext</code> and <code>aria-orientation</code> on an <code>input[type="range"]</code> control.
+      The desired temperature is set with the range control, which is vertically oriented.
+      The range control illustrate how to use <code>aria-valuetext</code> to provide assistive technologies with meaningful names for numeric values.
+    </p>
+    <p>Similar examples include: </p>
+    <ul>
+      <li><a href="slider-color-viewer.html">Color Viewer Slider Example</a>: Basic  horizontal sliders that illustrate setting numeric values for a color picker.</li>
+      <li><a href="examples/slider/slider-temperature.html">Temperature Selector Slider Example</a>: Demonstrates using <code>aria-orientation</code> to specify vertical orientation and <code>aria-valuetext</code> to communicate unit of measure for a temperature input.</li>
+      <li><a href="slider-rating.html">Rating Slider Example</a>: Horizontal slider that demonstrates using <code>aria-valuetext</code> to communicate current and maximum value of a rating input for a five star rating scale.</li>
+      <li><a href="multithumb-slider.html">Horizontal Multi-Thumb Slider Example</a>: Demonstrates using sliders with two thumbs to provide inputs for  numeric ranges, such as for searching in a price range.</li>
+    </ul>
+    <section>
+      <div class="example-header">
+        <h2 id="ex_label">Example</h2>
+      </div>
+      <div role="separator" id="ex_start_sep" aria-labelledby="ex_start_sep ex_label" aria-label="Start of"></div>
+      <div id="ex1"
+        class="example">
+          <div class="range-temperature">
+            <label for="id-temp-range">
+              Temperature
+            </label>
+            <div class="range-value">25.0°C</div>
+            <input type="range"
+              id="id-temp-range"
+              class="veritcal"
```

Comment:
typo in class name, and not actually used. just remove the whole `class="veritcal"` here

Reply:
Thanks, I will update